auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+        Args:
+            token: توکن
+            token_type: نوع توکن
+            user: کاربر
+            reason: دلیل
+            
+        Returns:
+            bool: موفقیت
+        """
+        try:
+            # محاسبه زمان انقضا
+            if token_type == 'access':
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "token_type"

(S105)